src/turbomind/kernels/bert_preprocess_kernels.cu

```diff
@@ -63,7 +63,7 @@ void invokeGetPaddingOffsetAndCuSeqLens(size_t*      h_pinned_token_num,
     h_pinned_token_num[0] = 0;
     getPaddingOffsetAndCuSeqLensKernel<<<1, 1, 0, stream>>>(
         h_pinned_token_num, tmp_mask_offset, cu_seqlens, sequence_lengths, batch_size, max_seq_len);
-    while (((volatile size_t*)h_pinned_token_num)[0] == 0) {};
```

Comment:
Why change it to `cudaStreamSynchronize`?

Reply:
will stuck on windows